### ISSUE TITLE
Wallpanel JS error on startup

### ISSUE THREAD

**Issue by j9brown** [j-a-n/lovelace-wallpanel#164]
After rebooting a tablet on which Wallpanel is running inside of the FullyKiosk browser, I noticed the following message in the Home Assistant logs and the Wallpanel screensaver did not display as expected.

It worked fine after a reload though.

Version: Wallpanel 4.18.0 running on Home Assistant 10.3

```
Logger: frontend.js.latest.202310050
Source: components/system_log/__init__.py:300
First occurred: 7:04:50 PM (1 occurrences)
Last logged: 7:04:50 PM

Uncaught error from Chrome WebView 117.0.0.0 on Android 13 TypeError: Cannot read properties of null (reading 'shadowRoot') /hacsfiles/lovelace-wallpanel/wallpanel.js:2179:11
```

**Comment by j-a-n** [comment]
Does this problem still exist with version 4.20.1?

**Comment by j9brown** [comment]
Uncertain.  I just updated and don't see the log message now.  I'll check again later.

**Comment by j-a-n** [comment]
Hopefully fixed with v4.22.1.

**Comment by j9brown** [comment]
I'm pretty sure this is fixed now.  I don't see the error in the log anymore in v4.23.  Thanks!
